runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/util/PackageUtilTest.java

```diff
@@ -195,7 +187,7 @@ public void testFileWithExtensionPackageNamingAndSize() throws Exception {
     PackageAttributes attr = makePackageAttributes(tmpFile, null);
     DataflowPackage target = attr.getDestination();
 
-    assertThat(target.getName(), RegexMatcher.matches("file-" + HASH_PATTERN + ".txt"));
+    assertThat(target.getName(), RegexMatcher.matches(UUID_PATTERN + ".txt"));
```

Comment:
You could just match `".*.txt"` here, rather than hard-coding the uuid format. (Same blow.)

Reply:
but it only checks whether the staged file name has the same extension (vs. checks whether the staged file name is in the form of UUID with the same extension)

Reply:
Is it important, for the purposes of this test (or Dataflow in general) that the staged file name is in the form of a UUID? If not (and I don't think it is) it's better not to test for it. (A test that might be good to add is to see if two same-named files in different directories actually get staged to different places, which is the underlying, important intent.)

Reply:
Hmm. You're right. UUID is implementation detail behind the uniqueness guarantee. I changed the tests to only check whether it keeps the same extension.